### DISCUSSION TITLE
How to create accountability object by token

### DISCUSSION THREAD

**Discussion by Enhed** [directus/directus#14549] [Archive]
I try to create microservice with tcp connection to directus, that app wil lbe send payload source token with other meta. So how i can create accountability object with that src token in extension?

**Comment by br41nslug** [comment]
There is generally no need to manually transform a token into accountability in extensions as that's handled by middleware for each request before the extensions are run. While I wouldn't recommend it you can build up the logic needed for this based on this code https://github.com/directus/directus/blob/main/api/src/middleware/authenticate.ts

**Reply by Enhed** [reply]
I cant use `import {verifyAccessJWT} from "directus/dist/utils/jwt"`
`ENOENT: no such file or directory, open node_modules\directus\dist\dist\utils\jwt.js` error in my extension
cannot understand why in path adding one more `dist`

**Reply by Enhed** [reply]
nvm i done, thx)
